paystell-frontend/src/components/dashboard/analytics/StellarAnalytics.tsx

```
@@ -0,0 +1,258 @@
+        startDate = startOfDay(subDays(now, 1)); // Last 24 hours might be better? Let's do last full day for simplicity now.
+        startDate = subDays(now, 1);
+        break;
+      case 'weekly':
+        startDate = startOfWeek(subDays(now, 7), { weekStartsOn: 1 }); // Start of last week (Monday)
+        startDate = subDays(now, 7);
+        break;
+      case 'monthly':
+        startDate = startOfMonth(subDays(now, 30)); // Start of last month
+        startDate = subDays(now, 30);
+        break;
+      case 'all':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)